File: app/src/main/java/com/android/keeper/recycle_items/NoteItem.java
package com.android.keeper.recycle_items;

public class NoteItem {

    private Integer note_id;
    private String note_title;
    private String note_description;

    public NoteItem(Integer note_id, String note_title, String note_description) {
        this.note_id = note_id;
        this.note_title = note_title;
        this.note_description = note_description;
    }

    public Integer getNote_id() {
        return note_id;
    }

    public String getNote_title() {
        return note_title;
    }

    public void setNote_title(String note_title) {
        this.note_title = note_title;
    }

    public String getNote_description() {
        return note_description;
    }

    public void setNote_description(String note_description) {
        this.note_description = note_description;
    }
}

File: app/src/main/java/com/android/keeper/recycle_items/NoteItem.kt
package com.android.keeper.recycle_items

import android.support.test.InstrumentationRegistry
import android.support.test.runner.AndroidJUnit4
import org.junit.runner.RunWith

class NoteItem(private val note_id: Int?, private var note_title: String?, private var note_description: String?) {
    fun getNote_id(): Int? {
        return note_id
    }

    fun getNote_title(): String? {
        return note_title
    }

    fun setNote_title(note_title: String?) {
        this.note_title = note_title
    }

    fun getNote_description(): String? {
        return note_description
    }

    fun setNote_description(note_description: String?) {
        this.note_description = note_description
    }

}